addons/bus/static/src/main_tab_service.js

```diff
@@ -0,0 +1,10 @@
+import { browser } from "@web/core/browser/browser";
+import { isIosApp } from "@web/core/browser/feature_detection";
+import { registry } from "@web/core/registry";
+import { mainTabLocalStorageService } from "@bus/main_tab_local_storage_service";
+import { electionWorkerService } from "@bus/main_tab_election_worker_service";
+
+export const mainTabService =
+    browser.SharedWorker && !isIosApp() ? electionWorkerService : mainTabLocalStorageService;
```

Comment:
Don't know why `!isIosApp()` there. Probably an old bug that's maybe not relevant now?
I would remove it and if someone complains we fix it. But note the ios app is deprecated therefore people should migrate to PWA anyway.